### PR TITLE
✨ Playground: Adapt sharing dialog to new fly-in design

### PR DESCRIPTION
**Pull request by robinvanopstal**

### CODE REVIEW

File: playground/src/share-view/share-view.hbs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<div class="share-view-intro">
+  <h4>Copy & paste unique URL</h4>
```
Comment (matthiasrohmer) [inline playground/src/share-view/share-view.hbs:2]: Use `&amp;`

File: playground/src/share-view/share-view.hbs [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+  <p>This unique URL contains all changes made up to that point. As you make further changes to the code, a new URL will be generated.</p>
+</div>
+<textarea id="share-url" class="share-view-url"></textarea>
```
Comment (matthiasrohmer) [inline playground/src/share-view/share-view.hbs:5]: Does it make sense to set this to `disabled`?

File: playground/src/share-view/share-view.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+import template from './share-view.hbs';
+
+import {Base64} from 'js-base64';
+import FlyIn from '../fly-in/fly-in.js';
+import * as Button from '../button/button.js';
+
+export function createShareView(editor) {
+  const target = document.getElementById('share-view');
+  const trigger = document.getElementById('share-toggle');
+  if (!target) return;
```
Comment (matthiasrohmer) [inline playground/src/share-view/share-view.js:25]: Couldn't you exit early? If there's no toggle there's no need to initialize the view, is there?

File: playground/src/share-view/share-view.js [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+  createShareUrl() {
+    const source = this.editor.getSource();
+    const shareUrl = new URL(window.location.href);
+    shareUrl.searchParams.delete('url');
+    shareUrl.hash = 'share=' + Base64.encode(source);
+
+    return shareUrl.toString();
+  }
+
+  onClick() {
```
Comment (matthiasrohmer) [inline playground/src/share-view/share-view.js:66]: `onClick` is ambiguous - use `onClickCopy`

File: playground/src/share-view/share-view.js [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+    this.textarea.select();
```
Comment (matthiasrohmer) [inline playground/src/share-view/share-view.js:67]: Does this work cross-browser? For the copy-script-button, we additionally use `setSelectionRange` but I can't remember why.

```
scriptTag.select();
scriptTag.setSelectionRange(0, 99999);
```

File: playground/src/share-view/share-view.hbs (hunk 2 above)
Comment (robinvanopstal) [reply playground/src/share-view/share-view.hbs:5]: Nope, I think `disabled` would not let you copy it's content to the clipboard ..? I've added `readonly` instead.

File: playground/src/share-view/share-view.js (hunk 5 above)
Comment (robinvanopstal) [reply playground/src/share-view/share-view.js:67]: as discussed

**Pull request merged**